### DISCUSSION TITLE
¿PAdES-LTV?

### DISCUSSION THREAD

**Discussion by pr-apes** [ctt-gob-es/clienteafirma#359] [Q&A]
Buenos días,

buscando en el proyecto, veo que hay un método `PdfUtil::enableLtv` y que incluso también se puede aplicar a sellado de tiempo (si no entiendo mal el código, cosa que es bastante posible).

La verdad es que sería muy útil poder tener firmas PDF longevas con AutoFirma. La he habilitado en Acrobat Reader y no puedo firmar con AutoFirma del mismo modo.

¿Se sabe cuándo se va a incluir en AutoFirma la posibilidad de firmas PAdES-LTV?

Muchas gracias y saludos.

**Comment by acuartango** [comment]
Hola!

Hay creada una Issue:
https://github.com/ctt-gob-es/clienteafirma/issues/289

A ver si entre tod@s podemos ayudar a promocionar esta importante funcionalidad para una próxima versión!

Gracias por comentar

**Reply by pr-apes** [reply]
Hola @acuartango,

conozco la incidencia (de hecho, es mío el comentario anterior a https://github.com/ctt-gob-es/clienteafirma/issues/289#issuecomment-1755723700).

Abrí esta pregunta porque entiendo que el sellado de tiempo es distinto de firmas longevas.

Da igual donde contesten. Desde luego, si el desarrollo principal y responder a todas las preguntas en este proyecto recae sobre una persona, entiendo que tarde mucho en contestar sobre cualquier caso (fallo, propuesta de mejora o cualquier pregunta).

Muchas gracias por el comentario.

**Reply by pr-apes** [reply]
@acuartango, creo que _AutoFirma 1.9_ firma por defecto con LTV si el almacén de certificados tiene la cadena completa (raíz e intermedios).
